Sources/StreamChatUI/CommonViews/CellSeperatorView/CellSeparatorCollectionViewLayout.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A reusable collection view layout that supports rendering separator views in the collection cells.
+open class CellSeparatorCollectionViewLayout<ExtraData: ExtraDataTypes>: UICollectionViewFlowLayout {
+
+    /// The height of the cell separator view.
+    open var separatorHeight: CGFloat = 0.5
+
+    /// The reuse identifier of the cell separator view.
+    open var separatorIdentifier = "CellSeparator"
+
+    /// Closure handler to decide what indexPaths's the separator should be hidden.
+    open var shouldHideSeparator: (IndexPath, UICollectionView) -> Bool = { _, _ in
+        return false
+    }
```

Comment:
What do you guys think about this? It is the most flexible API have come up with to hide the cells that the customer wants. An example to hide the last cell would be:
return indexPath.row == collectionView.numberOfItems(section: indexPath.section) - 1

Reply:
For me simple and cool, actually this looks pretty powerful at first sight. :D